tensorboard/uploader/uploader_test.py

```diff
@@ -342,7 +356,7 @@ def test_upload_server_error(self):
 
         # Of course a real Event stream will never produce the same Event twice,
         # but is this test context it's fine to reuse this one.
-        graph_event = event_pb2.Event(graph_def=bytes(950))
+        graph_event = event_pb2.Event(graph_def=_create_example_graph(950))
```

Comment:
1. Just so that I understand: some of the changes like these are not strictly necessary to fix the test, but are just so that we are using valid GraphDefs consistently right?

2. Previously the graph_def has an exact size of 950. But now it's >950. Do you think there is the risk of breaking something?

Reply:
1.) Correct.  2.) Also correct, and I was worried about this too because we test e.g. how many 100-byte chunks are needed to transmit the entire blob.  As it happens, the actual size is still less than 1000, so the number of chunks is still 10 and the tests work as is.  (If this had been an issue, I would have just reduced the attr size to 925 or something to compensate).